docs/openapi-issuer.yaml

```diff
@@ -233,6 +233,36 @@ paths:
         default:
           $ref: "#/components/responses/Unexpected"
 
+  /notification:
```

Comment:
Isn't the notification sent automatically?

Reply:
No, we have decided to make this operation manual in such a way that the issuer can receive an error message if the receipt of messages by the user is not enabled. We have switched to using only the standard service without opt-in.